### DISCUSSION TITLE
Create ERM for Databases

### DISCUSSION THREAD

**Discussion by Hypo-Privat** [hallvard/plantuml#155] [Ideas]
Hi i'm working on a solution for Databases without activate Referential Integrity. So a foreign key between tables is not defined. 

I can build up this by columns used in tables.

I works, but i like to have a vertical design an maybe a solution to group this by column name .

Would be nice you can help.

Thank you Gert

Looks like:
Primaty Table     referenz table    Colomn Name
 ACT              -->  ACT             :  ACTNO
 ACT              -->  EMPPROJACT      :  ACTNO
 ACT              -->  PROJACT         :  ACTNO
 ACT              -->  VACT            :  ACTNO
 ACT              -->  VEMPPROJACT     :  ACTNO
 ACT              -->  VPROJACT        :  ACTNO
 ACT              -->  VSTAFAC1        :  ACTNO
 ACT              -->  VSTAFAC2        :  ACTNO
 
 ' CATALOG 
 CATALOG          -->  CATALOG         :  NAME 
 CATALOG          -->  PRODUCT         :  NAME
 CATALOG          -->  STAFF           :  NAME
 CATALOG          -->  STAFFG          :  NAME
 CUSTOMER         -->  CUSTOMER        :  CID

 ...

**Comment by hallvard** [comment]
The (details of) layouting is handled by the plantuml library, I believe this is a problem that should be discussion in plantuml site.
